docs/antora/modules/ROOT/pages/configuration/source.adoc

```diff
@@ -9,254 +9,38 @@ Mona Bärenfänger <mona@lightcurve.io>
-// 			max_memory_restart: '300M',
-// 			autorestart: true,
-// 			env: {
-// 				// --- Remember to set the properties below
-// 				SERVICE_BROKER: 'redis://localhost:6379/0',
-// 				SERVICE_FEE_ESTIMATOR_CACHE: 'redis://localhost:6379/1',
-// 				ENABLE_FEE_ESTIMATOR_QUICK: 'true',
-// 				ENABLE_FEE_ESTIMATOR_FULL: 'false',
-// 			},
-// 		},
-// 		{
-// 			name: 'lisk-service-transaction-statistics',
-// 			script: 'app.js',
-// 			cwd: './services/transaction-statistics',
-// 			pid_file: './pids/service_transaction_statistics.pid',
-// 			out_file: './logs/service_transaction_statistics.log',
-// 			error_file: './logs/service_transaction_statistics.err',
-// 			log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
-// 			watch: false,
-// 			kill_timeout: 10000,
-// 			max_memory_restart: '300M',
-// 			autorestart: true,
-// 			env: {
-// 				// --- Remember to set the properties below
-// 				SERVICE_BROKER: 'redis://localhost:6379/0',
-// 				SERVICE_STATISTICS_REDIS: 'redis://localhost:6379/1',
-// 				SERVICE_STATISTICS_MYSQL: 'mysql://lisk:password@127.0.0.1:3306/lisk',
-// 				TRANSACTION_STATS_HISTORY_LENGTH_DAYS: '366',
-// 			},
-// 		},
-// 		{
-// 			name: 'lisk-service-market',
-// 			script: 'app.js',
-// 			cwd: './services/market',
-// 			pid_file: './pids/service_market.pid',
-// 			out_file: './logs/service_market.log',
-// 			error_file: './logs/service_market.err',
-// 			log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
-// 			watch: false,
-// 			kill_timeout: 10000,
-// 			max_memory_restart: '300M',
-// 			autorestart: true,
-// 			env: {
-// 				// --- Remember to set the properties below
-// 				SERVICE_BROKER: 'redis://localhost:6379/0',
-// 				SERVICE_MARKET_REDIS: 'redis://localhost:6379/2',
-// 				SERVICE_MARKET_FIAT_CURRENCIES: 'EUR,USD,CHF,GBP,RUB',
-// 				SERVICE_MARKET_TARGET_PAIRS: 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,BTC_EUR,BTC_USD,BTC_CHF',
-// 				// EXCHANGERATESAPI_IO_API_KEY: ''
-// 			},
-// 		},
-// 		{
-// 			name: 'lisk-service-export',
-// 			script: 'app.js',
-// 			cwd: './services/export',
-// 			pid_file: './pids/service_export.pid',
-// 			out_file: './logs/service_export.log',
-// 			error_file: './logs/service_export.err',
-// 			log_date_format: 'YYYY-MM-DD HH:mm:ss SSS',
-// 			watch: false,
-// 			kill_timeout: 10000,
-// 			max_memory_restart: '300M',
-// 			autorestart: true,
-// 			env: {
-// 				SERVICE_BROKER: 'redis://localhost:6379/0',
-// 				SERVICE_EXPORT_REDIS: 'redis://localhost:6379/3',
-// 				SERVICE_EXPORT_REDIS_VOLATILE: 'redis://localhost:6379/4',
-// 			},
-// 		},
-// 	],
-// };
-// ----
+:url_default_config: configuration/index.adoc#default-configurations
 
 == Custom configuration
+Lisk Service offers a set of xref:{url_default_config}[Default Configurations], which can be customized as per the requirement.
+To create a custom configuration, it is recommended to copy the default configuration file and use the copy of the said file for customization.
```

Comment:
The users can directly customize the existing `ecosystem.config.js` file!? 🤔

Reply:
Not sure, please suggest otherwise!

Reply:
I meant, they can update the environment variables in the existing `ecosystem.config.js` file instead of creating a copy.

Reply:
Do you want this sentence to be removed? And Should I suggest the reader to update the `ecosystem.config.js` file directly?

Reply:
Yes please.

Reply:
Although I agree with you on editing the default file, after giving it a thought I think the described way is safer in terms of having the default reference point available. This is something we do with the Lisk node config as well so that the default config stays as is. Having such files can be a go-to reference point for novice users so If you agree, we can keep it as is.

Reply:
Sure, that's fine by me. We can let it as is 👍